scripts/deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
I guess you want to call this `publish.sh` since that is what this does and what you are using in the circle scripts